content/python/concepts/math-module/terms/math-atan2/math-atan2.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: 'math.atan2()'
+Description: 'Takes two numeric values representing a point (x, y) and returns a float with the arc tangent of y divided by x. The resulting value will be in radians, between \pi and -\pi.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
```

Comment:
```suggestion
---
Title: 'math.atan2()'
Description: 'Returns the arc tangent of y/x in radians.'
Subjects:
  - 'Computer Science'
  - 'Data Science'
Tags:
  - 'Arithmetic'
  - 'Functions'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
---
```

Reply:
Description should be short, the other part can be added in the introduction paragraph below.